bin/update-snyk-pr.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const Github = require('./github')
+const SECURITY_PREFIX = 'security:'
+
+module.exports = async function updateSnykPR() {
```

Comment:
I would do something like this so you don't have to do that weird `node -e 'require` in the workflow.

```js
-module.exports = async function updateSnykPR() {
+async function updateSnykPR() {
   const org = process.env.RELEASE_ORG || 'newrelic'
   const repo = process.env.RELEASE_REPO || 'node-newrelic'
   const prId = process.env.SNYK_PR_ID
@@ -29,3 +29,10 @@ module.exports = async function updateSnykPR() {
 
   await github.updatePullRequest({ id: prId, title: newTitle })
 }
+
+if (require.main === module) {
+  updateSnykPR()
+} else {
+  module.exports = updateSnykPR
+}

```